lib/s3middleware/tagging.ts

```diff
@@ -106,6 +117,54 @@ function _validateTags(tags: Array<{ Key: string[], Value: string[] }>) {
     return tagsResult;
 }
 
+/** validateTags - Validate tags, throwing an error if tags are invalid
+* @param tags - tags parsed from xml to be validated
+* @param tags[].Key - Name of the tag
+* @param tags[].Value - Value of the tag
+* @return tagsResult - return object tags on success
+* { key: value }; error on failure
+*/
+export function validateTags(tags: Array<BucketTag>) {
+    if (tags.length === 0) {
+        return {};
+    }
+    // Maximum number of tags per resource: 50
+    if (tags.length > 50) {
+        return errorBadRequestLimit50();
+    }
+
+    const tagsResult = {};
+    for (const tag of tags) {
+        if (!_validator.validateTagObjectStructure(tag)) {
+            throw errors.MalformedXML;
+        }
+        const key = tag.Key;
+        const value = tag.Value;
+
+        if (!key) {
+            throw errors.InvalidTag.customizeDescription('The TagKey you ' +
+            'have provided is invalid');
```

Comment:
```suggestion
            throw errors.InvalidTag.customizeDescription(
                'The TagKey you have provided is invalid'
            );
```
I think this method of splitting lines is easier to read. It has the same column open and close. There is not a string starting on a normal code line, don't need to read two lines for a string. That sort of stuff.

Reply:
Removed as the condiiton was not needed anymore as you said.